pkg/lib/feature/sendpassword/template.go

```diff
@@ -21,6 +21,9 @@ type messageTemplateContext struct {
 var (
 	TemplateMessageChangePasswordTXT       = template.RegisterMessagePlainText("messages/change_password_email.txt")
 	TemplateMessageChangePasswordEmailHTML = template.RegisterMessageHTML("messages/change_password_email.html")
+
+	TemplateMessageCreateUserTXT       = template.RegisterMessagePlainText("messages/create_user_email.txt")
+	TemplateMessageCreateUserEmailHTML = template.RegisterMessageHTML("messages/create_user_email.html")
```

Comment:
`/messages/send_password_to_existing_user.html`
`/messages/send_password_to_new_user.html`

Reply:
Change the MessageType to

`send-password-to-existing-user`
`send-password-to-new-user`